cmd/tikv-ctl/src/main.rs

```diff
@@ -1080,6 +1169,39 @@ fn read_fail_file(path: &str) -> Vec<(String, String)> {
     list
 }
 
+struct TemporaryRocks {
+    rocks: RocksEngine,
+    #[allow(dead_code)]
+    tmp: TempDir,
+}
+
+impl TemporaryRocks {
+    fn new(cfg: &TikvConfig) -> Result<Self, String> {
+        let tmp = TempDir::new().map_err(|v| format!("failed to create tmp dir: {}", v))?;
```

Comment:
How about let the user decide the path of local rocksdb.

I'm concerned that the temp dir may not always be the fast disk in many environments.

Reply:
In fact we won't write anything to this RocksDB, this RocksDB instance is just used for loading the configuration for creating `SstWriter`.

Reply:
👌

Reply:
Please add some comments to this struct.